### DISCUSSION TITLE
Folders step 3: Folders UI

### DISCUSSION THREAD

**Discussion by kuzmany** [mautic/mautic#13087] [Announcements]
Based on https://docs.google.com/document/d/1nJxRDGslOg5aIxoLyK2RwGz1zek9i_DD9L555vnDHN4/edit 

Integrate folder structure to bundles to UI.

Should contain:

- folders on the top of the section of bundles from level 1 +  number of entities include in the folders
- last modified entities from all folders
- new column for table view folders/clickable labels  to certain folder

**Comment by RCheesley** [comment]
Pinging @mautic/ux-ui-tiger-team for their feedback/thoughts.
